content/20.treatments.md

```diff
@@ -571,4 +571,37 @@ Summarize the state of the neutralizing antibody approach.
+The majority of current clinical trials and lines of investigation have focused on repurposing existing therapies to counter SARS-CoV-2 and treat its symptoms.
+This is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
+However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
+There is thus value in investigating two lines of inquiry for treatment of COVID-19: 1) new therapeutics specific for treatment of COVID-19 or its symptoms, and 2) repurposing of existing therapeutics for treatment of COVID-19 or its symptoms.
+Here we consider further avenues that scientific investigators may explore in the development of therapies for COVID-19.
+
+Given the great focus in investigating hydroxychloroquine (HCQ) as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives.
+For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV [@doi:10.1021/jm0601856].
+Future work could explore whether these compounds exert antiviral effects against SARS-CoV-2 and if they are safe for use in animals and humans.
+
+The tocilizumab trial described in an above section [@doi:10/ggnxb3] studies the possibility of using an anti-inflammatory drug, typically used for the treatment of autoimmune disease, to counter the effects of the “cytokine storm” induced by the virus.
+Another anti-IL-6 antibody, sarilumab, is also being investigated [how to cite? http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19 or https://clinicaltrials.gov/ct2/show/NCT04327388].
+Typically, immunosuppressive drugs are often contraindicated in the setting of infection [@doi:10.1177/2040622313485275].
+However, when the need to curtail the “cytokine storm” inflammatory response to the virus transcends the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
+COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, suggesting that immunosuppression may be a helpful approach to treatment [@doi:10.1016/S0140-6736(20)30628-0].
+While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-alpha. Inhibition of TNF-alpha by an inhibitor such as Etanercept has been previously suggested for treatment of SARS-CoV [@doi:10.1185/030079903125002757] and may be relevant for SARS-CoV-2 as well.
+Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections [@doi:10.1016/S1473-3099(20)30132-8].
```

Comment:
```suggestion
Baricitinib and other small molecule inhibitors of the JAK kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections [@doi:10/dph5].
```
DOIs with parentheses cause a bug, so I used shortdoi.org to convert this -- your formatting looks great except for this rare case!

Reply:
Thanks for your help!